src/github.com/getlantern/http-proxy-extensions/mimic/apache.go

```diff
@@ -263,27 +271,27 @@ var badRequestBody = template.Must(template.New("notFound").Parse(
 var optionsHeader = template.Must(template.New("optionsHeader").Parse("HTTP/1.1 200 OK\r\n" +
 	"Date: {{.Date}}\r\n" +
 	"Server: Apache/2.4.7 (Ubuntu)\r\n" +
-	"Allow: POST,OPTIONS,GET,HEAD\r\n" +
+	"Allow: GET,HEAD,POST,OPTIONS\r\n" +
```

Comment:
The order changed from the test Apache server, so I made a change to pass the test. It's ok if we always return this header in fixed order.